inc/functions/files.php

```diff
@@ -1050,8 +1057,8 @@ function rocket_rrmdir( $dir, array $dirs_to_preserve = [] ) {
 	 *
 	 * @since 1.1.0
 	 *
-	 * @param string $dir              File/Directory to delete.
-	 * @param array  $dirs_to_preserve Directories that should not be deleted.
+	 * @param string       $dir              File/Directory to delete.
+	 * @param array|string $dirs_to_preserve Directories that should not be deleted.
 	 */
 	do_action( 'before_rocket_rrmdir', $dir, $dirs_to_preserve ); // phpcs:ignore WordPress.NamingConventions.PrefixAllGlobals
```

Comment:
@Tabrisrp Now that it's a regex pattern string (instead of an array), will it break something else?

Reply:
I don't think so, the actions are not used in the plugin. The other function using `rocket_rrmdir` with the second argument is `rocket_clean_user`, which uses the same format as `rocket_clean_domain`, so that should be working too

Reply:
Just to make sure, I reverted to keep that array and then created a new variable to hold the regex pattern. It's overkill. But at least it won't break anything that's hooked into that event. 2d33aff163d04f4e4c2160c420566b497a5af42d